code-of-conduct.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
+reported by contacting the project team at david.potesil@ceitec.muni.cz. All
```

Comment:
I was not sure what address to put there, but this one is probably the best one at this point, thanks